VideoApp/VideoApp/API/Requests/CommunityCreateTwilioAccessTokenRequest.swift

```diff
@@ -22,6 +22,7 @@ struct CommunityCreateTwilioAccessTokenRequest: APIRequest {
         let userIdentity: String
         let roomName: String?
         let createRoom: Bool
+        let createConversation = true
```

Comment:
Should this be settable in the initializer? Or is it always going to be `true`?

Reply:
It will always be true for us. And if a customer wants to remove chat they would set this to false and it would always be false for them. So there should never be a need to determine this at runtime.